app/timetables/models.py

```diff
@@ -123,6 +123,27 @@ def save(self, *args, **kwargs):
     def __str__(self):
         return self.name
 
+    def calculate_cycle_day(self, date_datetime):
+        days_interval = (date_datetime - self.ref_cycle_date).days
+
+        if days_interval < 0:
+            cycle_day = None
+        else:
+            cycle_day = ((days_interval % self.cycle_length) + self.ref_cycle_day) \
```

Comment:
@mfalade we do not have any coding convention against backslashes for wrapping long lines. Thanks for bringing that up. I agree with that. According to the style guide
```
The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. 
Long lines can be broken over multiple lines by wrapping expressions in parentheses. 
These should be used in preference to using a backslash for line continuation.
```
@andela-kerinoso please can we go with the recommendation on the style guide?